packages/instui-cli/lib/handlers/handleCreate.js

```diff
@@ -50,28 +50,9 @@ module.exports = async ({
     pkg.devDependencies
   )
 
-  if (contentType === 'app') {
-    mergeInDependencies(pkgDependencies, '@instructure/ui-babel-preset')
```

Comment:
The `mergeInDependencies` method at the bottom of the file is not used anywhere anymore, so it can be deleted too.